json/classic/src/test/java/ch/qos/logback/contrib/json/classic/JsonLayoutTest.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (C) 2016, The logback-contrib developers. All rights reserved.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ *
+ *   or (per the licensee's choosing)
+ *
+ * under the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation.
+ */
+package ch.qos.logback.contrib.json.classic;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.LoggingEvent;
+import ch.qos.logback.core.joran.spi.JoranException;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests the {@link JsonLayout} class
+ *
+ * @author Espen A. Fossen
+ */
+public class JsonLayoutTest {
+
+    private LoggerContext context = new LoggerContext();
+
+    private void configure(String file) throws JoranException {
+        JoranConfigurator jc = new JoranConfigurator();
+        jc.setContext(context);
+        jc.doConfigure(file);
+    }
+
+    @Test
+    public void addToJsonMap() throws Exception {
+        Map<String, Object> map = new LinkedHashMap<String, Object>();
+        JsonLayout jsonLayout = new JsonLayout();
+        jsonLayout.add("key1", true, "value1", map);
+        jsonLayout.add("key2", false, "value2", map);
+        jsonLayout.add("key3", true, null, map);
+
+        assertTrue(map.size() == 1);
```

Comment:
I typically prefer `assertThat` (w/Hamcrest) for [improved readability](https://objectpartners.com/2013/09/18/the-benefits-of-using-assertthat-over-other-assert-methods-in-unit-tests/) of tests and errors. Could you please rewrite? (should be trivial)